packages/@webex/contact-center/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "@webex/contact-center",
+  "description": "",
+  "main": "dist/index.js",
+  "devMain": "src/index.ts",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/webex/webex-js-sdk.git",
+    "directory": "packages/@webex/contact-center"
+  },
+  "engines": {
+    "node": ">=20.x"
+  },
+  "scripts": {
+    "build:src": "webex-legacy-tools build -dest \"./dist\" -src \"./src\" -js -ts && yarn build",
+    "build": "yarn run -T tsc --declaration true --declarationDir ./dist/types",
+    "deploy:npm": "yarn npm publish",
```

Comment:
Maybe we should remove this?

Reply:
Why should we remove this ?

Reply:
We do not want to publish this yet. Publishing should happen once we are code complete right?